apps/condo/domains/ticket/schema/UserFavoriteTicket.test.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Generated by `createschema ticket.UserFavoriteTicket 'user:Relationship:User:CASCADE; ticket:Relationship:Ticket:CASCADE;'`
+ */
+
+const dayjs = require('dayjs')
+
+const { UUID_RE, expectToThrowInternalError } = require('@open-condo/keystone/test.utils')
+const {
+    expectToThrowAccessDeniedErrorToObj,
+} = require('@open-condo/keystone/test.utils')
+
+const { UNIQUE_CONSTRAINT_ERROR } = require('@condo/domains/common/constants/errors')
+const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
+const { UserFavoriteTicket, createTestUserFavoriteTicket, updateTestUserFavoriteTicket, createTestTicket } = require('@condo/domains/ticket/utils/testSchema')
+
+describe('UserFavoriteTicket', () => {
+    let clientWithProperty1
+    let clientWithProperty2
+
+    beforeAll(async () => {
+        clientWithProperty1 = await makeClientWithProperty()
+        clientWithProperty2 = await makeClientWithProperty()
+    })
+
+    describe('Create', () => {
+        test('can create UserFavoriteTicket in user organization', async () => {
+            const [ticket] = await createTestTicket(clientWithProperty1, clientWithProperty1.organization, clientWithProperty1.property)
```

Comment:
you can move ticket creating in `beforeAll`
this happens in every test

Reply:
No, we have a unique user and ticket constraint on `UserFavoriteTicket` creation